installer/solr/pom.xml

```diff
@@ -1,9 +1,7 @@
-<project
-    xmlns="http://maven.apache.org/POM/4.0.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Unnecessary formatting changing make it challenging to determine if there are functional update buried in the noise. Please revert these updates.